import_flow_events.py

```diff
@@ -100,12 +101,15 @@
 """
 
 Q_CLEAR_DAY_METADATA = """
-    DELETE FROM flow_metadata
-    WHERE begin_time::DATE = '{day}'::DATE;
+    DELETE m
+    FROM flow_metadata AS m
+    LEFT JOIN flow_events AS e
+      ON m.flow_id = e.flow_id
+    WHERE e.flow_id IS NULL;
```

Comment:
I have a hard time understanding this query - is it "delete flow_metadata records that do not have any events"?

This raises the question of, how well do we cope with flows whose events are spread across the date boundary between two files?  I can imagine situations where we're deleting and re-import some-but-not-all of the events in a flow.